src/shipit_bot_uplift/requirements_frozen.txt

```diff
@@ -61,3 +61,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 whatthepatch==0.0.5
 yarl==0.10.2
+flake8-coding
```

Comment:
this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``